docs/website/02_examples/nonLinearElasticity.md

```diff
@@ -121,6 +126,30 @@ lc.subscribeAll(vtkWriter);
 lc.run();
 ```
 
+For postprocessing purposes we now write our results in a different VTK File. First we take a look at the stresses, in this case
+the second Piola-Kirchhoff stress tensor. This quantity can be computed with the elements' `calculateAt()` function.
+But here we will be using a so-called `ResultFunction`. This is a helper function that gathers the results over the
+whole grid and can be used to generate data for a `VTKWriter`. To create this function, we can use the following helper
+
+```cpp
+auto stressFunction = Ikarus::makeResultFunction<ResultType::PK2Stress>(&fes, req);
+```
+
+To add this as vertex data to the VTK file we can do the following:
+
+```cpp
+Dune::VTKWriter resultWriter(gridView);
+resultWriter.addVertexData(stressFunction);
+```
+
+As this functionality only writes out the results of the `calculateAt()` function as is, we can use `ResultEvaluators`
+to further process our results. For example, we can use `ResultEvaluators::VonMises<dim>` to compute the Von Mises stress:
```

Comment:
This PR looks ok for me but from the user perspective this call site can be more pretty. 
https://github.com/ikarus-project/ikarus/blob/d515373f013547881f09056000628d958d79ee24/ikarus/io/resultevaluators.hh#L27

The `VonMises` struct should not be a template bout only the operator(). Then yopu can pass only 

```cpp
  auto vonMisesFunction
      = Ikarus::makeResultFunction<ResultType::PK2Stress, ResultEvaluators::VonMises>(&fes, req);
```

Reply:
This does not need to be in this PR. But you can decide what you want to do.

Reply:
But sicne you touched the code also in examples it maybe less overhead to do this now.

Reply:
Yeah thats true, for now I am using the dim in the `()` function
https://github.com/ikarus-project/ikarus/blob/d515373f013547881f09056000628d958d79ee24/ikarus/io/resultevaluators.hh#L36

But i guess I could deduce this from the size of the result Array

Reply:
Ok it should work like that 
```cpp
 template <typename R>
  double operator()(const R& resultArray, [[maybe_unused]] const int comp) const {
    if constexpr (R::CompileTimeTraits::RowsAtCompileTime == 3) {
    ```

Reply:
I can do that in this PR